hack/common.sh

```diff
@@ -34,16 +34,21 @@ OSDK_WMCO_management() {
 
   local COMMAND=$1
   local OSDK_PATH=$2
+  local ARGS="packagemanifests \
+      --olm-namespace openshift-operator-lifecycle-manager \
+      --operator-namespace openshift-windows-machine-config-operator \
+      --operator-version 0.0.0"
 
+  # TODO: Remove retry logic hack once OSDK is upgraded to a version where 'operator-sdk run packagemanifests'
+  # works more reliably. Currently with v0.16.9 this hack is necessary as it often doesn't succeed on first attempt.
```

Comment:
We are using SDK version 0.19.4. v0.16.9 must be the OLM version.

Reply:
this is a typo, may bad! will push a fixup